kasa/modules/usage.py

```diff
@@ -21,6 +21,11 @@ def query(self):
 
         return req
 
+    @property
+    def query_response_size(self):
```

Comment:
```suggestion
    def estimated_query_response_size(self):
```
Let's rename this to avoid any potential confusion.